ember-cli-build.js

```diff
@@ -11,7 +11,10 @@ const env = EmberAddon.env() || 'development';
 module.exports = function (defaults) {
   console.log(`BUILD: Env is ${env}`);
   var app = new EmberAddon(defaults, {
-    // Add options here
+    // config for ember-cli-babel
+    'ember-cli-babel': {
+      includePolyfill: true
```

Comment:
We should **not** include the babel polyfill.

Reply:
nvm - this will _only_ get added to the addon's dummy app, so this is fine, _but_ I think it'd be best to add a comment 'splainin' why we did this, i.e. `// only needed so that the addon's dummy app can be run in IE`